frontend/src/pages/agreements/list/AgreementsTable.jsx

```diff
@@ -9,28 +9,32 @@ import { AgreementTableRow } from "./AgreementTableRow";
  * @returns {React.JSX.Element} - The rendered component.
  */
 export const AgreementsTable = ({ agreements = [] }) => {
-    return agreements.length > 0 ? (
-        <table className="usa-table usa-table--borderless width-full ">
-            <thead>
-                <tr>
-                    <th scope="col">Agreement</th>
-                    <th scope="col">Project</th>
-                    <th scope="col">Type</th>
-                    <th scope="col">Total</th>
-                    <th scope="col">Need By</th>
-                    <th scope="col" className="padding-0" style={{ width: "6.25rem" }}>
-                        Status
-                    </th>
-                </tr>
-            </thead>
-            <tbody>
-                {agreements?.map((agreement) => (
-                    <AgreementTableRow key={agreement?.id} agreement={agreement} />
-                ))}
-            </tbody>
-        </table>
-    ) : (
-        <div className="padding-1 text-bold">No agreements found.</div>
+    return (
+        <div>
+            <table className="usa-table usa-table--borderless width-full ">
```

Comment:
maybe don't show the table if there are no agreements?

Reply:
That was how it was implemented originally but the design was updated recently to show the table.